helix-term/src/commands.rs

```diff
@@ -1811,6 +1812,20 @@ fn select_all(cx: &mut Context) {
     doc.set_selection(view.id, Selection::single(0, end))
 }
 
+fn select_first_and_last_chars(cx: &mut Context) {
+    let (view, doc) = current!(cx.editor);
+
+    let selection = doc.selection(view.id).clone().transform_iter(|range| {
+        vec![
```

Comment:
```suggestion
        [
```

no need to create a Vec, we can iterate on a slice instead